src/sphinx_book_theme/header_buttons/launch.py

```diff
@@ -103,6 +103,7 @@ def add_launch_buttons(
     binderhub_url = launch_buttons.get("binderhub_url", "").strip("/")
     colab_url = launch_buttons.get("colab_url", "").strip("/")
     deepnote_url = launch_buttons.get("deepnote_url", "").strip("/")
+    retrolite_url = launch_buttons.get("retrolite_url", "").strip("/")
```

Comment:
```suggestion
    retrolite_url = launch_buttons.get("retrolite_url", "").rstrip("/")
```

jupyterlite needs to be an absolute URL (`/lite`) rather than relative to the current file. The 'strip' will strip out the leading / too, prevengin this from being an absolute URL